web_src/js/features/repo-branch.js

```diff
@@ -13,6 +21,13 @@ export function initRepoBranchButton() {
     }
 
     $(fromSpanName).text($(this).attr('data-branch-from'));
-    $($(this).attr('data-modal')).modal('show');
```

Comment:
This line can not be deleted. Otherwise you break the "create branch" modal on commit list page.